lib/vast/Conversion/Core/Passes.cpp

```diff
@@ -18,7 +18,7 @@ namespace vast::conv::pipeline {
         // This is necessary to have line tables emitted and basic debugger
         // working. In the future we will add proper debug information emission
         // directly from our frontend.
-        return nested< mlir::LLVM::LLVMFuncOp >(mlir::LLVM::createDIScopeForLLVMFuncOpPass)
+        return nested< vast_module >(mlir::LLVM::createDIScopeForLLVMFuncOpPass)
```

Comment:
why this?

Reply:
Because upstream changed what the pass operates on (it is on module now) - the upstream commit message had some reasoning why this happened.